chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -828,6 +828,7 @@ sync.localDataWillBeCombined		= If you continue, local Zotero data will be combi
 sync.localGroupsWillBeRemoved1		= Local groups, including any with changed items, will also be removed from this computer.
 sync.avoidCombiningData				= To avoid combining data, revert to the '%S' account or use the Reset options in the Sync pane of the Zotero preferences.
 sync.unlinkWarning					= Are you sure you want to unlink this account?\n\n%S will no longer sync your data, but your data will remain locally.
+sync.warning.emptyLibrary			= Your are about to sync to an empty %1$S database. This might be due to a new installation %1$S or the location of your data directory changing.\n\nDo you want to continue?
 
```

Comment:
Ah yes, I wrote this string when I thought I'd check with the remote repository whether there were any items there and haven't updated for logic changes.